### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by chrisr-diffblue**
Add two new code owners for goto-analyzer and goto-instrument @danpoe and
@hannes-steffenhagen-diffblue. Also add @danpoe as an owner for src/analyses,
and add @chrisr-diffblue as an owner for .travis.yml

- [x] Each commit message has a non-empty body, explaining why the change was made.
- [ ] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [ ] The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [ ] My commit message includes data points confirming performance improvements (if claimed).
- [x] My PR is restricted to a single feature or bugfix.
- [ ] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

**Review by smowton — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -62,5 +62,5 @@
 /scripts/delete_failing_smt2_solver_tests
 /scripts/expected_doxygen_warnings.txt
 
-/.travis.yml @diffblue/devops @thk123 @forejtv @peterschrammel
+/.travis.yml @diffblue/devops @thk123 @forejtv @peterschrammel @chrisr-diffblue
 /appveyor.yml @diffblue/devops @thk123 @forejtv @peterschrammel
```
Comment (tautschnig) [inline CODEOWNERS]: Would you mind removing that `appveyor.yml` line (possibly in a separate commit)?

**Review by tautschnig — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (chrisr-diffblue) [reply CODEOWNERS]: Sure!

**Comment by chrisr-diffblue** [comment]
Also added code owners for the new goto-harness tool in a separate commit.

**Review by peterschrammel — APPROVED** [review]

**Review by allredj — COMMENTED** [review]
:heavy_check_mark:
Passed Diffblue compatibility checks (cbmc commit: 7c540575c1ae9cefab069e1a6b718fbaf86c5b14).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/100123790

**Review by allredj — COMMENTED** [review]
:heavy_check_mark:
Passed Diffblue compatibility checks (cbmc commit: b969f0307ff7b758984aa29a6945e2e010b3f9bf).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/100126150

**Pull request merged**